### DISCUSSION TITLE
Play/pause button and progress bar stopped showing

### DISCUSSION THREAD

**Discussion by abelonogova** [katspaugh/wavesurfer.js#3020] [Q&A] [answered]
The issue is occurring on this page https://www.energyogre.com/why-electric-bill-is-high and other similar pages that are using the plugin for implementing audio. 

To import the plugin we're using the url reference:
```
<script src="https://unpkg.com/wavesurfer.js"></script>
```

The following code is used in addition to the plugin reference:
```
<script> 
  $( document ).ready(function() {
  var wavesurfer = WaveSurfer.create({ 
  container: document.querySelector('#wave'),
  backend: 'MediaElement', 
  mediaControls: true, 
  responsive: true, 
  height : 0, 
  barHeight:5,
  autoCenter: true,
  hideScrollbar: true,
  progressColor :'#97C94C',
  waveColor: '#437485', 
  minPxPerSec:1,
  autoCenter:true,
  scrollParent:true,
  barRadius:5, 
  cursorWidth: 2, 
  cursorColor:'#437485'});
wavesurfer.load("https://eo-upload.s3.amazonaws.com/blog-audios/Blog+Audio+Why+Is+My+Electric+Bill+So+High.mp3");
  });



</script>
```

In the code of the page the `#shadow-root` element is being added, yet the buttons to play/pause the record are not displaying.

The code hasn't been changed in any way and the audio seemed to work fine before, but at some point it stopped displaying the elements.
What could be the issue?

![Screenshot 2023-07-14 101937](https://github.com/katspaugh/wavesurfer.js/assets/100177228/faff795c-b03a-4942-92f6-6f51d387f7a2)
![Screenshot 2023-07-14 102008](https://github.com/katspaugh/wavesurfer.js/assets/100177228/49bf7ca0-18f8-4b30-9bae-9b85166ee51c)
![Screenshot 2023-07-14 102436](https://github.com/katspaugh/wavesurfer.js/assets/100177228/b95b7cd8-502e-4d53-9f58-94dad683dd97)

**Comment by katspaugh** [comment]
You should replace `height: 0` with some actual value, or remove it completely.

**Reply by abelonogova** [reply]
@katspaugh thank you for your response!
I tried giving the height property a value and it displayed the progress bar this way:

![Screenshot 2023-07-18 073056](https://github.com/katspaugh/wavesurfer.js/assets/100177228/bcdafca2-7c70-4236-b5a6-a924d73cecf3)


though before we used to have the following view with the play/pause btns, simple line progress bar, volume, etc:

![Screenshot 2023-07-18 073112](https://github.com/katspaugh/wavesurfer.js/assets/100177228/4201633b-e882-48a4-81d3-7d2bf4bb0059)

So I'm trying to figure out if it had something to do with the plugin, and if it allowed such a view before, but not anymore?

**Reply by katspaugh** [reply]
That’s the default audio element. You don’t need wavesurfer if that’s what you’re after. Just use the standard audio HTML tag.

**Reply by abelonogova** [reply]
@katspaugh yes, that's what I was thinking too.
Just out of curiosity, is there a way to achieve that with wavesurfer?

**Reply by katspaugh** [reply, marked as answer]
It’s possible, yes.
You could add `@6` to the unpkg URL, so your old config would work.
I think I’ll restore the mediaContrlols option in v7 too, which is the reason it stopped working for you.

**Reply by abelonogova** [reply]
@katspaugh perfect, that worked for me! thank you so much!
is there a way for me to subscribe to the plugin releases so not to forget to check for the mediaContrlols option?

**Reply by katspaugh** [reply]
Great!
I’ve created an issue for it: https://github.com/katspaugh/wavesurfer.js/issues/3025

Thank you for the feedback!

**Reply by abelonogova** [reply]
@katspaugh awesome! thanks so much!!

**Reply by abelonogova** [reply]
@katspaugh hey, just wanted to check, how often do you guys usually release the updates for plugin to production? I just noticed you merged the updates above to the master branch, but when checking my website, I saw that the issue was still there if referencing the latest plugin version, so I figured you guys haven't released it to prod yet?
thank you!

**Reply by katspaugh** [reply]
Minor versions every week or so.
I've just released 7.0.4 with that fix and a couple others.

**Reply by abelonogova** [reply]
Perfect! Thanks so much!!
